src/zmalloc.h

```diff
@@ -62,11 +62,18 @@
 #endif
 
 /* On native libc implementations, we should still do our best to provide a
- * HAVE_MALLOC_SIZE capability.
+ * HAVE_MALLOC_SIZE capability. This can be set explicitly as well:
```

Comment:
any reason not to support that explicit macro with jemalloc too?

Reply:
Just for the sake of simplicity.

Reply:
i think some (rare) users may want to explicitly disable for jemalloc too.

Reply:
Right now it satisfies package maintainers on non-jemalloc platforms and testing needs, I think modifying the code is justified for anything else (and if we come across a good reason we can add that).

The feature/platform detection matrix and resulting defines is complex enough, I'd really want to avoid any additional complexity. Having the temptation to pull `autoconf` in is not a good sign 😄